packages/salesforcedx-utils-vscode/src/telemetry/telemetry.ts

```diff
@@ -79,12 +79,12 @@ export class TelemetryService {
   public async initializeService(
     extensionContext: ExtensionContext,
     extensionName: string,
-    aiKey: string,
+    aiKey: string | undefined,
```

Comment:
```suggestion
    aiKey = extensionPackage.aiKey,
```

alternatively you could just use the string value but I think leaving it in the package json of sfdx-utils-vscode is the way to go.